roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -45,28 +38,43 @@ backup_driver = cinder.backup.drivers.swift
 backup_swift_url = http://{{ endpoints.swift.haproxy_vip }}:{{ endpoints.swift.port.cinder_backup }}/v1/
 {% endif -%}
 
-{% if ceph.enabled|bool -%}
-{% if cinder.common_volume_name -%}
-host = ceph
-{% endif -%}
+enabled_backends =
+{%- for backend, enabled in [('lvm', lvm.enabled),
+                            ('rbd_hybrid', ceph.enabled and ceph_pools.rbd_hybrid.enabled),
+                            ('rbd_ssd', ceph.enabled and ceph_pools.rbd_ssd.enabled)] if enabled -%}
+{{ backend + ',' }}
+{%- endfor %}
+
 
 {% if cinder.default_volume_type | default(false) -%}
 default_volume_type = {{ cinder.default_volume_type }}
 {% else %}
-default_volume_type = {{ ceph_pools[ceph_default_backend]['volume_type'] }}
+{% for volume_type, enabled in [('LVM', lvm.enabled),
+                                (ceph_pools[ceph_default_backend]['volume_type'] if ceph.enabled else '', ceph.enabled)] if enabled %}
+{% if loop.last %}
+default_volume_type = {{ volume_type }}
 {% endif %}
-
-{% if ceph_pools.rbd_ssd.enabled and ceph_pools.rbd_hybrid.enabled -%}
-enabled_backends = rbd_hybrid,rbd_ssd
-{% elif ceph_pools.rbd_ssd.enabled -%}
-enabled_backends = rbd_ssd
-{% elif ceph_pools.rbd_hybrid.enabled -%}
-enabled_backends = rbd_hybrid
+{% endfor%}
 {% endif %}
 
+{% if lvm.enabled -%}
+[lvm]
+volume_backend_name = lvm
+volume_driver = cinder.volume.drivers.lvm.LVMVolumeDriver
+iscsi_helper=tgtadm
+iscsi_ip_address = {{ primary_ip }}
+volume_group = cinder-volumes
+use_forwarded_for = true
+{% endif -%}
+
+{% if ceph.enabled|bool -%}
+
 {% for key, value in ceph_pools.iteritems() %}
 {% if value.enabled %}
 [{{ key }}]
+{% if cinder.common_volume_name -%}
+host = ceph
```

Comment:
shouldn;t host variable be outside of [key] .. it is under [default] block in conf and under specific backend block. Please verify against working stack and ensure we not moving things to wrong place.